src/Illuminate/Contracts/Routing/ResponseFactory.php

```diff
@@ -7,7 +7,7 @@ interface ResponseFactory
     /**
      * Create a new response instance.
      *
-     * @param  string  $content
+     * @param  mixed  $content
```

Comment:
This is incorrect. It's okay to widen a param type in the implementation. The facade change is okay though.

Reply:
Do I need to undo this change?

Reply:
Yes please revert this part.